incubator/distributed-tensorflow/templates/statefulset-worker.yaml

```diff
@@ -0,0 +1,88 @@
+{{- $lr := .Values.hyperparams.learningrate -}}
+{{- $batchsize := .Values.hyperparams.batchsize -}}
+
+apiVersion: apps/v1beta2
+kind: StatefulSet
+metadata:
+  name: {{ .Release.Name }}-worker
+  labels:
+    app: {{ template "distributed-tensorflow.name" . }}
+    chart: {{ template "distributed-tensorflow.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    role: worker
+spec:
+  selector:
+    matchLabels:
+      app: {{ template "distributed-tensorflow.name" . }}
+      chart: {{ template "distributed-tensorflow.chart" . }}
+      release: {{ .Release.Name }}
+      heritage: {{ .Release.Service }}
+      role: worker
+  serviceName: {{ template "distributed-tensorflow.fullname" . }}
+  podManagementPolicy: {{ .Values.worker.podManagementPolicy }}
+  replicas: {{.Values.worker.number}}
+  template:
+    metadata:
+      labels:
+        app: {{ template "distributed-tensorflow.name" . }}
+        chart: {{ template "distributed-tensorflow.chart" . }}
+        release: {{ .Release.Name }}
+        heritage: {{ .Release.Service }}
+        role: worker
+    spec:
+{{- if .Values.volumes }}
+      volumes:
+{{ toYaml .Values.volumes | indent  6 }}
+{{- end }}
+      containers:
+      - name: worker
+        image: "{{ .Values.worker.image.repository }}:{{ .Values.worker.image.tag }}"
+        imagePullPolicy: {{ .Values.worker.image.pullPolicy }}
+        command:
+          - "python"
+          - "train_distributed.py"
+{{- if gt (int .Values.worker.gpuCount) 0 }}
+          - --num_gpus
+          - "{{ .Values.worker.gpuCount }}"
+{{- end }}
+          - --learning_rate
+          - {{ $lr | quote }}
+          - --batch_size
+          - {{ $batchsize | quote }}
+        env:
+         - name: WORKER_HOSTS
```

Comment:
Use a consistent indentation style throughout the chart, either zero or two spaces.